### ISSUE TITLE
Consider renaming/relocating /usr/lib64/pm-utils/sleep.d/52qubes-pause-vms

### ISSUE THREAD

**Issue by mirrorway** [QubesOS/qubes-issues#3909]
### Qubes OS version:

R4.0

### Affected component(s): 

qubes suspend handling

---

### Steps to reproduce the behavior:

1. want to install a sleep hook
2. notice qubes calls /usr/lib64/pm-utils/sleep.d/52qubes-pause-vms
3. great, I'll just create a sleep hook called 40my-sleep-hook and stick it there
4. suspend

### Expected behavior:

40my-sleep-hook runs before 52qubes-pause-vms

### Actual behavior:

40my-sleep-hook does not run.

### General notes:

It turns out /usr/lib/systemd/system/qubes-suspend.service directly runs 52qubes-pause-vms, and nothing else in that folder.

So sleep hooks have to be set up via systemd service file. Which is fine, but the current setup is deceptive.

---

**Comment by marmarek** [comment]
Yes, I think pm-utils is nowadays completely dead and replaced by systemd handling (at least on Fedora), so we can abandon this legacy location.
